rest_framework/authtoken/models.py

```diff
@@ -40,14 +41,31 @@ def __str__(self):
         return self.key
 
 
-class TokenProxy(Token):
-    """
-    Proxy mapping pk to user pk for use in admin.
-    """
-    @property
-    def pk(self):
-        return self.user.pk
+if 'rest_framework.authtoken' in settings.INSTALLED_APPS:
```

Comment:
The token model could also be moved into this block and use the same pattern as I did for `TokenProxy`